blivet/zfcp.py

```diff
@@ -253,9 +251,9 @@ def online_device(self):
         unitdir = "%s/%s" % (portdir, self.fcplun)
         failed = "%s/failed" % (unitdir)
 
-        # Activating an NPIV enabled device using devnum, WWPN and LUN should still be possible
-        # as this method was used as a workaround until the support for NPIV enabled devices has
-        # been implemented. Just log a warning message and continue.
+        # Activating using devnum, WWPN, and LUN despite available zFCP auto LUN scan should still
+        # be possible as this method was used as a workaround until the support for NPIV enabled
+        # devices has been implemented. Just log a warning message and continue.
```

Comment:
# be possible as this method was used as a workaround until the support for zfcp auto LUN scan
        # has been implemented. Just log a warning message and continue.

[Sorry, I think I had missed that before.]

Reply:
Fixed.